### DISCUSSION TITLE
Returning avg_points from player class

### DISCUSSION THREAD

**Discussion by joshsjf** [cwendt94/espn-api#396] [ESPN API] [answered]
Hi, API looks brilliant and am would love to start using it. 
I'm trying to return the avg_points for players but am unable to do so. Is there something I am missing? I'm using v0.26.1 of the API. 
[This discussion suggests that avg_points has been implemented since v0.19.0](https://github.com/cwendt94/espn-api/discussions/281)
Thanks :)
![image](https://user-images.githubusercontent.com/41312161/200441621-84ad4e28-e0a9-4e79-a05a-909b3f0944a6.png)

**Comment by cwendt94** [comment, marked as answer]
Just looked into this and found a bug! This was recently broken when adding more stats to the player class. Fixing now and will be in v0.26.2

**Comment by joshsjf** [comment]
I pulled down v0.26.2 and it seems the bug still persists

**Comment by joshsjf** [comment]
Have just found 'stats' and it provides all of the categories I was looking for, rather than just average points as I'm presuming 'avg_points' returns? 
![image](https://user-images.githubusercontent.com/41312161/200654356-bf0b49a6-4301-4e8e-963e-525bd84df31d.png)

**Reply by cwendt94** [reply]
Yeah stats has all of the raw data and breakdown, then the `avg_points` and `total_points` field pulls from the stats. The assumption is that `applied_total` and `applied_avg` should be valued but you can always fall back on the `avg` `PTS` field and `total` `PTS` field in the stats dict.
